starttls-policy/starttls_policy/configure.py

```diff
@@ -79,6 +86,9 @@ class PostfixGenerator(ConfigGenerator):
     """
 
     def _generate(self, policy_list):
+        if util.is_expired(policy_list.expires):
+            self.expired_warning()
+            return "# Policy list is outdated. Falling back to opportunistic encryption."
```

Comment:
Since the expired warning should probably be the same across different config generators, let's move this check & report up to `ConfigGenerator.generate`?

However, the resulting "fallback" may change depending on the MTA, so maybe we can add an abstract `_generate_expired_fallback` that returns the "empty fallback" contents for that particular MTA.

I'm open to other ideas on this too. What do you think?